README.md

```diff
@@ -72,6 +74,14 @@ To simplify searches, this API also has provided the feature of provided the fun
 
 Ctrl('r') is the way to invoke that functionality.
 
+# General attributes
+
+1. | (pipe operator)
+The above command will combine multiple command and run them sequentially.
+
+Sample Syntax:
+
+listDir -a|listDir -l
```

Comment:
Spacing before and after |

Reply:
Works both ways. With or without spacing before pipe. But will update it in the REARDME